master/usage/migrating-pools.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Changing IP pools
+---
+
+## About changing IP pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured IP pools.
+There are a few reasons you may want to change the IP pool being used on a running cluster, including moving to a
+larger CIDR which can accomodate more workloads, or moving off of a CIDR that was used accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to change from one IP pool to another on a running cluster.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Orchestrator support**
+
+While {{ site.prodname }} supports changing IP pools, not all orchestrators do.
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it.
```

Comment:
I think you should lose the "for whichever" and maybe just go with "of" . Also, it's unclear what the second "it" refers to. In addition, you might also have a few unnecessary words in this sentence. I might suggest something like:

```
Consult the documentation of your installer to ensure it supports changing IP pools.
```

I am also wondering, is it the documentation of the installer or the orchestrator? If it is the orchestrator, as I suspect, then I would replace "installer" with "orchestrator".

Reply:
Note that most installers / orchestrators don't have a concept of an IP pool - they are likely to call it something different.